test/definitions/native.jl

```diff
@@ -280,13 +280,14 @@ module LazyCodegen
         # Note this follows: emit_call_specfun_other
         JuliaContext() do ctx
```

Comment:
```suggestion
        JuliaContext() do ts_ctx
```

Reply:
`JuliaContext()` is still a regular context, right?